_episodes/04-lists.md

```diff
@@ -160,32 +160,42 @@ does not.
 > ## Nested Lists
 > Since a list can contain any Python variables, it can even contain other lists.
 >
-> For example, we could represent the products in the shelves of a small grocery shop:
+> For example, you could represent the products on the shelves of a small grocery shop
+> as a nested list called x:
+>
+> [![x is represented as a shelf full of produce. There are four rows of vegetables
+> on the shelf, and each row contains five baskets of vegetables. We can label
+> each basket according to the type of vegetable it contains, so the top row
+> contains (from left to right) lettuce, lettuce, peppers, zucchini, squash,
+> and basil.](../fig/04_groceries_x.png)]
+>
+> To store the contents of the shelf in a nested list, you write it this way:
 >
 > ~~~
-> x = [['pepper', 'zucchini', 'onion'],
->      ['cabbage', 'lettuce', 'garlic'],
->      ['apple', 'pear', 'banana']]
+> x = [['lettuce', 'lettuce', 'peppers', 'zucchini', 'squash', 'basil'],
+>      ['lettuce', 'lettuce', 'peppers', 'zucchini', 'peppers', 'parsley'],
+>      ['lettuce', 'cilantro', 'peppers', 'zucchini', 'squash', 'spinach'],
+>      ['cabbage', 'broccoli', 'asparagus', 'zucchini', 'squash', 'cauliflower']]
 > ~~~
 > {: .language-python}
 >
-> Here is a visual example of how indexing a list of lists `x` works:
+> Here are some visual examples of how indexing a list of lists `x` works. First,
+> you can reference each row on the shelf as a separate list. For example, x[2]
+> represents the list of baskets on the third row of the shelf.
```

Comment:
Let's, perhaps, say, that `x[2] represents {proper qualifier} row, which is, in turn, a list of baskets...`